Lesson16-tugba/script.js

```diff
@@ -0,0 +1,370 @@
+/*
+8. Print Each Character of a String (For-of)
+   - Define a function `printCharacters(str)` that uses a for-of loop
+     to log each character in the string on a separate line.
+*/
+
+function printCharacters(str) {
+  for (char of str) {
```

Comment:
Don't forget to add keyword 'const' when defining a variable in for...in / for...of loops. Without it compiler will try to figure out what to do and behavior may not always be as expected. 

```suggestion
  for (const char of str) {
```